pkg/operator/ceph/object/user/controller_test.go

```diff
@@ -67,7 +67,7 @@ const (
 	"caps": [
 		{
 			"type": "users",
-			"perms": "*"
+			"perm": "*"
```

Comment:
changing this to be `read`, may avoid changes needed in the test case. I am unsure whether to add logs in the unit tests for debugging purposes, so please remove them. And you can add a test case that tests updating caps for a user

Reply:
Even if the perm is changed to `read`, the test case still needs to be modified, because removing user caps was skipped before, and remove user caps will be executed after this fix. Otherwise, [setting Capabilities for the user](https://github.com/rook/rook/blob/10abbd0d8a0ff2e5330ceaca733a31eec4391a5a/pkg/operator/ceph/object/user/controller_test.go#L448) and [setting both Quotas and Capabilities for the user](https://github.com/rook/rook/blob/10abbd0d8a0ff2e5330ceaca733a31eec4391a5a/pkg/operator/ceph/object/user/controller_test.go#L498) have covered updating caps for a user.